config_tuab_eeg.py

```diff
@@ -1,15 +1,14 @@
-from sys import path
-from pathlib import Path
+# from sys import path
+# from pathlib import Path
 import mne
 
 study_name = "age-prediction-benchmark"
 
 # On drago
-N_JOBS = 80
-bids_root = Path("/storage/store2/data/TUAB-healthy-bids-bv")
-#deriv_root = Path("/storage/store3/derivatives/TUAB-healthy-bids2")
-deriv_root = Path("/storage/store3/derivatives/TUAB-healthy-bids3")
-subjects_dir = Path('/storage/store/data/camcan-mne/freesurfer')
+n_jobs = 80
+bids_root = "/storage/store2/data/TUAB-healthy-bids-bv"
+deriv_root = "/storage/store3/derivatives/TUAB-healthy-bids3"
+subjects_dir = "/storage/store/data/camcan-mne/freesurfer"
```

Comment:
Why would you do that?

Sure, it's  not necessary to pass `Path` objects, but is this change really necessary?